db/migrate/20191206182459_create_access_control_entry.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAccessControlEntry < ActiveRecord::Migration[5.2]
+  def change
+    create_table :access_control_entries do |t|
+      t.string :group_uuid
+      t.bigint :tenant_id
+      t.string :permission
+      t.references :aceable, :name => :access_control_entries, :polymorphic => true, :index => true
```

Comment:
shouldn't you index `group_uuid`, `permission` and `aceable_type` together because of the way they are searched?

Reply:
@bzwei done